content/applications/websites/blog.rst

```diff
@@ -0,0 +1,117 @@
+
+- :guilabel:`Name`: your tag’s name;
+- :guilabel:`Category`: the category your tag belongs to. You can select an existing one, create a
+  new one, or leave it empty;
+- :guilabel:`Used in`: to apply your tag to existing blog posts, click :guilabel:`Add a line`,
+  select your posts, and click :guilabel:`Select`. Click :guilabel:`New` if your tag applies to a
+  post you want to create on the fly.
+
+.. note::
+   To show tags on your Blog dashboard page, click :menuselection:`Edit --> Customize`, and toggle
+   the switch of :guilabel:`Sidebar` and then of :guilabel:`Tags List`.
+
+Customizing content
+===================
+
+You can enable the features related to your blog dashboard or a specific blog post according to your
+needs.
+
+Customizing your blog dashboard
+-------------------------------
+
+From your Blog dashboard page, click :menuselection:`Edit --> Customize`, and toggle the switch of
+the following options to activate them:
+
+:guilabel:`Top Banner`: select :guilabel:`Name/Latest Post` to display the name of the latest post
+on the top banner, or :guilabel:`Drop Zone for Building Blocks` to hide the top banner.
+
+:guilabel:`Layout`: select :guilabel:`Grid` or :guilabel:`List`.
+
+- :guilabel:`Cards`: to display your posts as cards;
+- :guilabel:`Increase Readability`: to improve the readability of the posts that are displayed.
+
+:guilabel:`Sidebar`: when enabled, additional options are available:
+
+- :guilabel:`Archives`: a search bar allows to search posts by creation date;
+- :guilabel:`Follow Us`: invites users to follow you on social media;
+- :guilabel:`Tags List`: shows all existing tags related to your posts. Users can click on a tag
+  to access all related posts.
+
+:guilabel:`Posts List`: select :guilabel:`Cover` to show the illustration related to your post, or
+:guilabel:`No Cover` to hide it.
+
+- :guilabel:`Author`: to show the author of the post;
+- :guilabel:`Comments/View Stats`: to show the number of comments and views on   your post;
+- :guilabel:`Teaser & Tags`: to show a teaser text and tags on your posts.
+
+Customizing your blog post
+--------------------------
+
+Click :menuselection:`Edit --> Customize` and toggle the switch of the following options to activate
+them:
+
+:guilabel:`Layout`: select :guilabel:`Title Inside Cover` or :guilabel:`Title above Cover`.
+
+- :guilabel:`Increase Readability`: to improve the readability of your text.
+
+:guilabel:`Sidebar`: to display the :guilabel:`Sidebar` and additional options:
+
+- :guilabel:`Archive`: a search bar allows to search posts by creation date;
+- :guilabel:`Author`: to mention the author on the post;
+- :guilabel:`Blog List`: to show the list of your blogs;
+- :guilabel:`Share Links`: to allow users to share your post on social media;
+- :guilabel:`Tags`: to display the tags of the post.
+
+:guilabel:`Breadcrumb`: to show the path to the post and the user's location.
```

Comment:
Perhaps: and the user's location within the website.
Just so it's clear what kind of location we mean ;)